cinema/views.py

```diff
@@ -56,3 +110,19 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+
+class OrderViewSet(viewsets.ModelViewSet):
+    queryset = Order.objects.all().prefetch_related(
+        "tickets__movie_session",
```

Comment:
This line is redundant, because tickets__movie_session__movie will also prefetch tickets__movie_session and tickets